integrations/LinkedInInsightTag/browser.js

```diff
@@ -30,5 +35,32 @@ class LinkedInInsightTag {
     logger.debug("=== in isReady LinkedIn Insight Tag===");
     return !!window._linkedin_data_partner_id;
   }
+
+  track(rudderElement) {
+    logger.debug("===In LinkedIn Insight Tag Track===");
+    const { message } = rudderElement;
+    const { event } = message;
+    if (!event) {
+      logger.error(
+        "[LinkedIn Insight Tag]: Event name is missing for track call!!==="
+      );
+      return;
+    }
+    const trimmedEvent = event.trim();
+    const eventMapping = getHashFromArray(
```

Comment:
1-1 mapping is supported or 1 - many is supported. The code is written for 1-1

Reply:
Yes, 1-1 mapping is supported.

Reply:
I'll update it with 1 to many.

Reply:
I'll update it with 1 to many.